File: .sdk/tm/js/test/exists.test.js

const { test, describe } = require('node:test')
const { equal } = require('node:assert')


const { MixpanelDataPipelinesSDK } = require('..')


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await MixpanelDataPipelinesSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- MixpanelDataPipelines SDK exists test

local sdk = require("mixpanel-data-pipelines_sdk")

describe("MixpanelDataPipelinesSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
